code/modules/projectiles/guns/projectile/revolver/longboi.dm

```diff
@@ -1,6 +1,7 @@
 /obj/item/gun/projectile/revolver/longboi
 	name = "\"Long\" revolver"
-	desc = "A novelty revolver made by the Lonestar with a stupidly long barrel favored by those with less sense than style, its extended barrel gives it incredible recoil control but loses stopping power and penetration. Uses 12mm."
+	desc = "A novelty revolver made by the Lonestar with a stupidly long barrel favored by those with more sense and style, its extended barrel gives it incredible recoil control but for some reason loses stopping power and penetration. \
+	Shouldn't it be the other way around? Uses 12mm."
```

Comment:
```suggestion
	Uses 12mm."
```

Reply:
The the whole flavour text should be removed because its wrong.

A long barrel on a revolver increases accuracy and velocity and thus more stopping power and penetration while making recoil more controllable.

Either the revolver needs to be re-coded or the flavour text removed altogether because its physically wrong.